### PR TITLE
fix wording in custom.py

### PR DESCRIPTION
**Pull request by yucwan**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by yucwan** [comment]
One problem identified but not fixed in this pr:

The error:
Deployment <name> not found, use 'az asc app deployment create' to create a new deployment
is currently only in app deploy, should be applied to all app level command with -d parameters.

File: src/asc/azext_asc/custom.py [hunk 1]
```diff
@@ -228,17 +229,17 @@ def app_get_log(cmd, client, resource_group, service, name, deployment=None):
...
 
 def app_set_deployment(cmd, client, resource_group, service, name, deployment):
     deployments = _get_all_deployments(client, resource_group, service, name)
     active_deployment = client.apps.get(resource_group, service, name).properties.active_deployment_name
     if deployment == active_deployment:
-        raise CLIError("deployment '" + deployment + "' is alredy the active deployment")
+        raise CLIError("Deployment '" + deployment + "' is already the production deployment")
     if deployment not in deployments:
-        raise CLIError("deployment '" + deployment + "' not found, please use 'az asc app deploy create' to create new deployment first")  
+        raise CLIError("Deployment '" + deployment + "' not found, please use 'az asc app deploy create' to create new deployment first")
```
Comment (yucwan) [inline src/asc/azext_asc/custom.py]: 'az asc app deployment create'

File: src/spring-cloud/azext_spring_cloud/tanzu.py [hunk 2]
```diff
@@ -148,8 +151,55 @@ def tanzu_app_deploy(cmd, client, resource_group, service, name, artifact_path,
...
+        raise CLIError("Failed to get a SAS URL to upload context. Error: {}".format(e.message))
+    # upload file
+    if not upload_url:
+        raise CLIError("Failed to get a SAS URL to upload context.")
+    account_name, endpoint_suffix, share_name, relative_name, sas_token = get_azure_files_info(upload_url)
+    logger.warning("[2/3] Uploading package to blob")
+    path = _get_tanzu_upload_local_file()
+    file_service = FileService(account_name, sas_token=sas_token, endpoint_suffix=endpoint_suffix)
+    file_service.create_file_from_path(share_name, None, relative_name, path)
+    if target_module is not None:
```
Comment (yuwzho) [inline src/spring-cloud/azext_spring_cloud/tanzu.py]: nit:
```python
        properties = models.BuildProperties(
            builder="default-tanzu-builder",
            name=name,
            relative_path=relative_path,
            env={"BP_JVM_VERSION": "8.*", "BP_MAVEN_BUILT_MODULE": target_module}) if target_module else None
```

File: src/spring-cloud/azext_spring_cloud/tanzu.py [hunk 3]
```diff
@@ -148,8 +151,55 @@ def tanzu_app_deploy(cmd, client, resource_group, service, name, artifact_path,
...
+    logger.warning("[3/3] Creating or Updating build '{}' (this operation can take a while to complete)".format(name))
+    build_result_id = None
+    try:
+        response = client.build_service.create_or_update_build(resource_group, service, name, properties)
+        build_result_id = response.properties.additional_properties["triggeredBuildResult"]["id"]
+        build_result_name = parse_resource_id(build_result_id)["resource_name"]
+    except (AttributeError, CloudError) as e:
+        raise CLIError("Failed to create or update a build. Error: {}".format(e.message))
+    # get build result
+    logger.warning(" - Building source code..")
```
Comment (yuwzho) [inline src/spring-cloud/azext_spring_cloud/tanzu.py]: `logger.warning("[4/5] Waiting build finish")` Let's keep the warning style align

File: src/spring-cloud/azext_spring_cloud/_params.py [hunk 4]
```diff
@@ -56,7 +56,8 @@ def load_arguments(self, _):
         c.argument('env', env_type)
     
     with self.argument_context('spring-cloud tanzu app deploy') as c:
-        c.argument('artifact-path', help='artifact path to deploy to this deployment')
+        c.argument(
+                'target_module', help='Child module to be deployed, required for multiple jar packages built from source code.')
```
Comment (yuwzho) [inline src/spring-cloud/azext_spring_cloud/_params.py:61]: update helper.py file as well?

**Pull request merged**